{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 6,
    "last_updated": "2025-06-12T20:12:45.480071",
    "severity_distribution": {
      "critical": 0,
      "high": 4,
      "medium": 1,
      "low": 0,
      "none": 1
    }
  },
  "cves": [
    {
      "id": "CVE-2025-5485",
      "publishedDate": "2025-06-12T20:05:35.216Z",
      "lastModifiedDate": "2025-06-12T20:05:35.216Z",
      "description": "User names used to access the web management interface are limited to \nthe device identifier, which is a numerical identifier no more than 10 \ndigits. A malicious actor can enumerate potential targets by \nincrementing or decrementing from known identifiers or through \nenumerating random digit sequences.",
      "severity": 8.6,
      "references": [
        {
          "url": "https://www.cisa.gov/news-events/ics-advisories/icsa-25-160-01",
          "type": "reference"
        },
        {
          "url": "https://www.sinotrackgps.com/help-center",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "IOT PC Platform",
          "vendor": "SinoTrack",
          "versions": [
            {
              "status": "affected",
              "version": "All versions"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-204"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-5484",
      "publishedDate": "2025-06-12T20:03:32.255Z",
      "lastModifiedDate": "2025-06-12T20:06:27.184Z",
      "description": "A username and password are required to authenticate to the central \nSinoTrack device management interface. The username for all devices is \nan identifier printed on the receiver. The default password is \nwell-known and common to all devices. Modification of the default \npassword is not enforced during device setup. A malicious actor can \nretrieve device identifiers with either physical access or by capturing \nidentifiers from pictures of the devices posted on publicly accessible \nwebsites such as eBay.",
      "severity": 8.3,
      "references": [
        {
          "url": "https://www.cisa.gov/news-events/ics-advisories/icsa-25-160-01",
          "type": "reference"
        },
        {
          "url": "https://www.sinotrackgps.com/help-center",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "IOT PC Platform",
          "vendor": "SinoTrack",
          "versions": [
            {
              "status": "affected",
              "version": "All versions"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-1390"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-6031",
      "publishedDate": "2025-06-12T19:29:11.082Z",
      "lastModifiedDate": "2025-06-12T20:01:01.630Z",
      "description": "Amazon Cloud Cam is a home security camera that was deprecated on December 2, 2022, is end of life, and is no longer actively supported. \n\nWhen a user powers on the Amazon Cloud Cam, the device attempts to connect to a remote service infrastructure that has been deprecated due to end-of-life status. The device defaults to a pairing status in which an arbitrary user can bypass SSL pinning to associate the device to an arbitrary network, allowing for network traffic interception and modification.\n\nWe recommend customers discontinue usage of any remaining Amazon Cloud Cams.",
      "severity": 7.5,
      "references": [
        {
          "url": "https://aws.amazon.com/security/security-bulletins/AWS-2025-013/",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "Cloud Cam",
          "vendor": "Amazon",
          "versions": [
            {
              "lessThanOrEqual": "*",
              "status": "affected",
              "version": "0",
              "versionType": "custom"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-672 Operation on a Resource after Expiration or Release"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-44019",
      "publishedDate": "2025-06-12T19:51:56.663Z",
      "lastModifiedDate": "2025-06-12T20:06:50.361Z",
      "description": "AVEVA PI Data Archive products are vulnerable to an uncaught exception that, if \nexploited, could allow an authenticated user to shut down certain \nnecessary PI Data Archive subsystems, resulting in a denial of service. \nDepending on the timing of the crash, data present in snapshots/write \ncache may be lost.",
      "severity": 7.1,
      "references": [
        {
          "url": "https://www.cisa.gov/news-events/ics-advisories/icsa-25-162-07",
          "type": "reference"
        },
        {
          "url": "https://www.aveva.com/en/support-and-success/cyber-security-updates/",
          "type": "reference"
        },
        {
          "url": "https://my.osisoft.com/",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "PI Data Archive",
          "vendor": "AVEVA",
          "versions": [
            {
              "lessThanOrEqual": "2018 SP3 Patch 4",
              "status": "affected",
              "version": "0",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Data Archive",
          "vendor": "AVEVA",
          "versions": [
            {
              "status": "affected",
              "version": "2023"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Data Archive",
          "vendor": "AVEVA",
          "versions": [
            {
              "status": "affected",
              "version": "2023 Patch 1"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Server",
          "vendor": "AVEVA",
          "versions": [
            {
              "lessThanOrEqual": "2018 SP3 Patch 6",
              "status": "affected",
              "version": "0",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Server",
          "vendor": "AVEVA",
          "versions": [
            {
              "status": "affected",
              "version": "2023"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Server",
          "vendor": "AVEVA",
          "versions": [
            {
              "status": "affected",
              "version": "2023 Patch 1"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-248"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-36539",
      "publishedDate": "2025-06-12T19:56:33.320Z",
      "lastModifiedDate": "2025-06-12T20:04:23.575Z",
      "description": "AVEVA PI Data Archive products \nare vulnerable to an uncaught exception that, if exploited, could allow \nan authenticated user to shut down certain necessary PI Data Archive \nsubsystems, resulting in a denial of service.",
      "severity": 6.5,
      "references": [
        {
          "url": "https://www.cisa.gov/news-events/ics-advisories/icsa-25-162-07",
          "type": "reference"
        },
        {
          "url": "https://www.aveva.com/en/support-and-success/cyber-security-updates/",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "product": "PI Data Archive",
          "vendor": "AVEVA",
          "versions": [
            {
              "lessThanOrEqual": "2018 SP3 Patch 4",
              "status": "affected",
              "version": "2023",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Data Archive",
          "vendor": "AVEVA",
          "versions": [
            {
              "status": "affected",
              "version": "2023 Patch 1"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Server",
          "vendor": "AVEVA",
          "versions": [
            {
              "lessThanOrEqual": "2018 SP3 Patch 6",
              "status": "affected",
              "version": "2023",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "product": "PI Server",
          "vendor": "AVEVA",
          "versions": [
            {
              "lessThanOrEqual": "2018 SP3 Patch 6",
              "status": "affected",
              "version": "2023 Patch 1",
              "versionType": "custom"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-248"
      ],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-48699",
      "publishedDate": "2025-06-12T20:12:40.833010+00:00",
      "lastModifiedDate": "2025-06-12T20:06:31.278Z",
      "description": "",
      "severity": "N/A",
      "references": [],
      "affected": [],
      "problemType": [],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}